docs/manual-gce-deployment.md

```diff
@@ -0,0 +1,374 @@
+# Deploying Gluster-Kubernetes, S3, and CNS Object Broker Manually in GCE
+
+## Limitations
+
+- Kubernetes is deployed via the `kubeadm` tool.  This is a work in progress deployer with its own [limitations](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#limitations).
+
+## Assumptions
+- A [Google Cloud Platform](https://cloud.google.com/) account.
+- 4 Running RHEL 7.2 GCE Instances.
```

Comment:
Will rhel image later than 7.2 work? Can we say 7.2+ ?

Reply:
It's untested, but i would guess yes